### PR TITLE
add some spacing to full width image

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Type

🐛 Bugfix  

## Value
Makes things look nice.

## Screenshot
__Before__
standalone image screengrab

__After__
screen shot 2017-07-24 at 16 42 22

### All
- [x] Demoed to the relevant people
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### CODE REVIEW

File: client/scss/components/_body_content.scss [hunk 1]
```diff
@@ -101,6 +102,7 @@ $center-width-xlarge: (6/6) * 100%;
     margin-right: calc((#{$narrow-width} + #{map-get($gutter-width, 'large')}) * -1);
     padding: 0 map-get($container-padding, 'large');
     padding-top: $vertical-space-unit;
+    padding-bottom: $vertical-space-unit * 2;
```
Comment (jamesgorrie) [inline client/scss/components/_body_content.scss]: I could put the spacing classes in here - but that would be a bit of a refactor, I'd rather just this for now.

Potentially in the future we could have something like `spacingClassesFor(el)`?

File: client/scss/components/_body_content.scss [hunk 2]
```diff
@@ -91,6 +91,7 @@ $center-width-xlarge: (6/6) * 100%;
   left: 50%;
   margin-left: -50vw;
   padding: 0 map-get($container-padding, 'small');
+  background: color('cream');
```
Comment (davidpmccormick) [inline client/scss/components/_body_content.scss]: I have vague memories of removing this as it had been causing problems elsewhere – maybe the transporters at the bottom of the digital stories?

Comment (davidpmccormick) [inline client/scss/components/_body_content.scss]: screen shot 2017-07-24 at 16 50 12

Comment (jamesgorrie) [inline client/scss/components/_body_content.scss]: You are 100% correct.

Comment (jamesgorrie) [inline client/scss/components/_body_content.scss]: I've redone this so it only applies to images that are standalone...

**Pull request merged**